src/qmapshack/gis/trk/CTrkToRteDialog.cpp

```diff
@@ -0,0 +1,97 @@
+/**********************************************************************************************
+    Copyright (C) 2021 Oliver Eichler <oliver.eichler@gmx.de>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+**********************************************************************************************/
+
+#include "canvas/CCanvas.h"
+#include "CMainWindow.h"
+#include "gis/CGisDatabase.h"
+#include "gis/CGisListWks.h"
+#include "gis/db/CDBProject.h"
+#include "gis/db/CSelectDBFolder.h"
+#include "gis/db/CSetupFolder.h"
+#include "gis/gpx/CGpxProject.h"
+#include "gis/qms/CQmsProject.h"
+#include "helpers/CSettings.h"
+#include "gis/trk/CTrkToRteDialog.h"
```

Comment:
Get rid of include hell. As far as I can see you only use:

```
#include "canvas/CCanvas.h"
#include "CMainWindow.h"
#include "gis/CGisWorkspace.h"
#include "helpers/CSettings.h"
#include "gis/trk/CTrkToRteDialog.h"
```

Reply:
My intention was to set the explanatory text to a smaller size. This looks good at my setup but obviously this had not the intended effect. I have checked the qt documentation, but I haven't found an other portable way, so I will reset the dialog to it's defaults.